<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1217</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPS explorer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_files_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>451</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="title">
     <string>Load SMLM data </string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QComboBox" name="comboBox_fileformat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>39</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
     <property name="currentText">
      <string>Picasso hdf5</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_filename">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>File format</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Filename</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_browsefile">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>40</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_vis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>731</width>
      <height>611</height>
     </rect>
    </property>
    <property name="title">
     <string>Visualization (output)</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>251</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="scatterlayout"/>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>30</y>
       <width>241</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="zhistlayout"/>
    </widget>
    <widget class="QRadioButton" name="radioButtonxz">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>330</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot (x,z)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_smallROI">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Small ROI</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="scatterlayout_3"/>
    </widget>
    <widget class="QRadioButton" name="radioButtonxy">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot (x,y)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonyz">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Plot (y,z)</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>360</y>
       <width>281</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="zhistlayout_2"/>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_scatter">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Scatter</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1217</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
